public/content/translations/de/developers/docs/networks/index.md

```diff
@@ -56,12 +56,11 @@ Die beiden öffentlichen Testnets, die die Client-Entwickler derzeit betreiben,
 - [QuickNode Sepolia Faucet](https://faucet.quicknode.com/drip)
 - [Grabteeth](https://grabteeth.xyz/)
 - [PoW-Faucet](https://sepolia-faucet.pk910.de/)
-- [Sepolia-Faucet](https://faucet.sepolia.dev/)
-- [FaucETH](https://fauceth.komputing.org)
-- [Coinbase Wallet Faucet | Sepolia](https://coinbase.com/faucets/ethereum-sepolia-faucet)
-- [Alchemy Sepolia-Faucet](https://sepoliafaucet.com/)
-- [Infura Sepolia-Faucet](https://www.infura.io/faucet)
-- [Chainstack Sepolia-Faucet](https://faucet.chainstack.com/sepolia-faucet)
+- [Faucet für Coinbase-Wallet | Sepolia](https://coinbase.com/faucets/ethereum-sepolia-faucet)
```

Comment:
The use of für here seems weird... 

Literally "Faucet for Coinbase-Wallet"

@lukassim any thoughts?